docs/howtos/customisations/embeddings.ipynb

```diff
@@ -0,0 +1,380 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "0174eb96",
+   "metadata": {},
+   "source": [
+    "# Using different Embedding Models\n",
+    "\n",
+    "Ragas allows users to change the default embedding model used in the evaluation task.\n",
+    "\n",
+    "This guide will show you how to use different embedding models for evaluation in Ragas."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "55f0f9b9",
+   "metadata": {},
+   "source": [
+    "## Evaluating with Open AI Embeddings\n",
+    "\n",
+    "Ragas uses BAAI/bge-small-en-v1.5 by default. In this example we can use Open AI Embeddings from langchain which uses text-embedding-ada-002. We will be using gpt4 as the llm for evaluation and `AnswerSimilarity` as the metric\n",
```

Comment:
Ragas uses OpenAI embeddings by default. Instead of open-ai it would be better to show azure-open-ai here